internal/app/app.go

```diff
@@ -142,31 +142,39 @@ func envVarOverrides(v *viper.Viper, cfg *Configuration) error {
 	}
 
 	if cfg.FleetDBOpts.AudienceEndpoint == "" {
-		return errors.New("fleetdb client secret not defined")
+		return errors.New("fleetdb OIDC audience endpoint not defined")
+	}
+
+	if v.GetString("fleetdb.issuer.endpoint") != "" {
+		cfg.FleetDBOpts.IssuerEndpoint = v.GetString("fleetdb.issuer.endpoint")
+	}
+
+	if cfg.FleetDBOpts.IssuerEndpoint == "" {
+		return errors.New("fleetdb OIDC issuer endpoint not defined")
 	}
 
 	if v.GetString("fleetdb.client.id") != "" {
 		cfg.FleetDBOpts.ClientID = v.GetString("fleetdb.client.id")
 	}
 
 	if cfg.FleetDBOpts.ClientID == "" {
-		return errors.New("fleetdb client id not defined")
+		return errors.New("fleetdb OIDC client id not defined")
 	}
 
 	if v.GetString("fleetdb.client.secret") != "" {
```

Comment:
Do we also want to add an env  `COMPONENT_INVENTORY_FLEETDB_CLIENT_SECRET placeholder` in `deployment.yaml`?

Reply:
My thought was to leave that one empty as a nudge to remind people to use an environment variable instead. If we add a placeholder it might be easier for someone to forget and commit a secret to source control? Also, I don't know if you can populate a config map entry with a secret the way you can do that with an environment variable.

I don't know, it made sense to me when I did it. 😅